homeassistant/components/overkiz/select.py

```diff
@@ -0,0 +1,119 @@
+"""Support for Overkiz select."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HomeAssistantOverkizData
+from .const import DOMAIN, IGNORED_OVERKIZ_DEVICES
+from .entity import OverkizDescriptiveEntity
+
+
+@dataclass
+class OverkizSelectDescriptionMixin:
+    """Define an entity description mixin for select entities."""
+
+    options: list[str]
+    select_option: Callable[[str, Callable], None]
```

Comment:
```suggestion
    select_option: Callable[[str, Callable], Awaitable]
```

Reply:
Thanks, I was totally looking in the wrong direction! Fixed :)